### DISCUSSION TITLE
How to check if channel allows links in comments

### DISCUSSION THREAD

**Discussion by imlordeyaya** [LuanRT/YouTube.js#141] [General]
Hello, Im looking for any way how to programaticaly find if channel allows links in comments or not. Do you think it is possible through API? I did not find this info in official API. Is it possible with this tool? Thanks for any help or suggestions.

**Comment by LuanRT** [comment]
As far as I can tell it is not possible through InnerTube either.

**Reply by imlordeyaya** [reply]
Ok thanks for help, at least I know I need to find another way.
